index.html

```diff
@@ -125,17 +127,18 @@ <h3 class="project-name">Tonic</h3>
             <div class="card grid-item card-2">
                 <div class="image-4"></div>
                 <div class="card-info">
-                    <h3 class="project-name">Multi-Post Stories</h3>
+                    <h3 class="project-name">Uber Navigation</h3>
                     <div class="project-info">
-                        <div class="company">CANOPY</div>
+                        <div class="company">Uber</div>
                         <div class="dot"></div>
-                        <div class="position">Back End Dev</div>
+                        <div class="position">Lead Developer</div>
                         <div class="dot"></div>
-                        <div class="year">2015</div>
```

Comment:
You didd well with the desktop version of your portfolio, kindly reduce the width of your portfolio project button to fit with the design given 😄 : 

Your design :
![image](https://github.com/ayaderaghul/microverse-portfolio/assets/20537712/83ebee84-644d-4238-965f-0de621709c26)

Figma design:
![image](https://github.com/ayaderaghul/microverse-portfolio/assets/20537712/d2cc0da0-5d11-443f-a516-b7784208df7d)